microprofile/testing/testng/pom.xml

```diff
@@ -35,9 +35,14 @@
 
     <dependencies>
         <dependency>
-            <groupId>io.helidon.microprofile.cdi</groupId>
-            <artifactId>helidon-microprofile-cdi</artifactId>
-            <scope>provided</scope>
+            <groupId>io.helidon.microprofile.server</groupId>
```

Comment:
There must still be a dependency on `helidon-microprofile-cdi` in provided scope (and keep the server dependency as optional)